step-automation-packages/step-automation-packages-junit/src/test/java/step/client/StepRunnerYamlPlanTest.java

```diff
@@ -19,11 +19,16 @@
 package step.client;
 
 import org.junit.runner.RunWith;
+import step.automation.packages.junit.AutomationPackagePlans;
 import step.handlers.javahandler.AbstractKeyword;
 import step.junit.runner.Step;
+import step.junit.runners.annotations.ExecutionParameters;
 import step.junit.runners.annotations.Plans;
 
 @RunWith(Step.class)
-@Plans({"composite-simple-plan.yml"})
+@Plans({"plans/composite-simple-plan.yml"})
```

Comment:
@david-stephan 
1. Now we have to specify both `@AutomationPackagePlans` (to choose the plans to be executed in test) and `@Plans` (to configure plans to be scanned in AP Reader). It may be not clear for user. Maybe we can rename the `@AutomationPackagePlans` or provide some documentation to explain how it should be used?

2. Is it OK that `@Plans` is now not fully compatible to old versions (plans are now referenced basing on root folder but not on current package)?